### PR TITLE
Dw 1265 create plan price

### PR DESCRIPTION
**Pull request by jcamposmk**
### **Create Plan Price component**
**Details task:** https://makingsense.atlassian.net/browse/DW-1265

- add price with discount
- add price per extra email

### CODE REVIEW

**Review by silsanchez8 — COMMENTED** [review]
Great Job Junior! 💪

File: src/components/Plans/PlanCalculator/PlanPrice/PricePerExtraEmail/index.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        <p className="dp-cost-per-email">
+          {_('plan_calculator.cost_per_email')}{' '}
+          <span className="dp-price-large-money">
+            <strong>US${extraEmailPrice}</strong>
+          </span>
+        </p>
+      ) : (
+        <></>
+      )}
+    </>
```
Comment (andresmoschini) [inline src/components/Plans/PlanCalculator/PlanPrice/PricePerExtraEmail/index.js:27]: What do you think about returning early and reducing nesting?

```suggestion
  if (selectedPlan.type !== PLAN_TYPE.byEmail && selectedPlan.type !== PLAN_TYPE.byCredit) {
    return (<></>);
  }

  return (
    <p className="dp-cost-per-email">
      {_('plan_calculator.cost_per_email')}{' '}
      <span className="dp-price-large-money">
        <strong>US${extraEmailPrice}</strong>
      </span>
    </p>
  );
```

Comment (jcamposmk) [reply src/components/Plans/PlanCalculator/PlanPrice/PricePerExtraEmail/index.js:27]: :+1:

**Pull request merged**